provd/debian/control

```diff
@@ -0,0 +1,18 @@
+Source: provd
+Section: admin
+Priority: optional
+Maintainer: Matt <matt.hagemann@canonical.com>
+Build-Depends: debhelper-compat (= 13)
+Standards-Version: 4.5.1
+Homepage: https://github.com/canonical/ubuntu-desktop-provision/tree/main/provd
+Rules-Requires-Root: no
+Vcs-Git: https://github.com/canonical/ubuntu-desktop-provision/tree/main/provd.git
+Vcs-Browser: https://github.com/canonical/ubuntu-desktop-provision/tree/main/provd
+
+Package: provd
+Architecture: any
+Multi-Arch: foreign
+Depends: ${misc:Depends}, ${shlibs:Depends}
```

Comment:
I would add one dependencies by line, also this package isn’t multi-arch, and have a trailing comma :)